cluster-controller/src/test/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperationsTest.java

```diff
@@ -81,13 +113,76 @@ public void testCreateCluster(TestContext context) {
                 .endMetadata()
                 .withData(cmData)
                 .build();
-        kubeClient.configMaps().create(clusterCm);
+        when(mockCmOps.get(clusterCmNamespace, clusterCmName)).thenReturn(clusterCm);
```

Comment:
Any idea how we could test that the operations are done in some order? We do not need it now, but as discussed, we might need it in the future.

Reply:
Good question. Mockito has a method `then()` which we could use instead of `thenReturn()` to do arbitrary things in addntion to returning the method result or throwing. For example we could record the time at which the method (on the resource `*Operation`) was called and I suppose we could also introduce things like `Thread.sleep()` so thing running concurrently finish a different times.

Reply:
Ok, good ... something like that should be feasible.